### ISSUE TITLE
Accessibility Audit: Improper label of close button in My account dashboard

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#937]
Solved with https://github.com/bloom-housing/bloom/issues/2249

Turn on the Screen Reader, and log in to the public portal. On the My Account page, there is a welcome message with a close button on the right. When you focus on the close button, instead of the Screen Reader calling it 'close', it announces it as 'times' (it seems to register as a multiplication symbol instead of an X). This should be announced as a close button instead.

https://user-images.githubusercontent.com/394908/147755755-ba927005-846d-479e-916c-ca8b8127097f.mov

**Comment by slowbot** [comment]
@kathyccheng We are waiting until the alert box component updates from core make it to Detroit. @seanmalbert
